Feature/rpnv1 439 validator url

flotothemoon opened the pull request.

Commit:
Add url field to RegisteredValidatorParticle

Commit:
Validate url and add tests for that

clstrfsck commented on src/test/java/com/radixdlt/atommodel/validators/ValidatorConstraintScryptTest.java:
```diff
@@ -89,6 +89,22 @@ public void when_validating_registered_validator_particle_with_no_address__resul
+		RegisteredValidatorParticle validator = mock(RegisteredValidatorParticle.class);
+		when(validator.getAddress()).thenReturn(mock(RadixAddress.class));
+		when(validator.getUrl()).thenReturn("http://www.radixdlt.com/");
+		staticCheck.apply(validator);
```
Should the result be checked here to ensure it has no error?

flotothemoon replied:
It should! Good catch. I forgot that we had switched from exceptions to return values here some time ago.

clstrfsck reviewed (changes requested):
Looks like a check of the result is missing from one of the unit tests.

Commit:
Fix ValidatorConstraintScryptTest for valid URL

Need to check the result of the check since we no longer throw exceptions on error
